tests/test_mlkit.py

```diff
@@ -185,29 +230,47 @@
 invalid_string_or_none_args = [0, -1, 4.2, 0x10, False, list(), dict()]
 
 
+# For validation type errors
 def check_error(err, err_type, msg=None):
-    assert isinstance(err, err_type)
+    err_value = err.value
+    assert isinstance(err_value, err_type)
     if msg:
-        assert str(err) == msg
+        assert str(err_value) == msg
+
+
+# For errors that are returned in an operation
+def check_operation_error(err, code, msg):
+    err_value = err.value
+    assert isinstance(err_value, exceptions.FirebaseError)
+    assert err_value.code == code
+    assert str(err_value) == msg
 
 
+# For rpc errors
 def check_firebase_error(err, code, status, msg):
-    assert isinstance(err, exceptions.FirebaseError)
-    assert err.code == code
-    assert err.http_response is not None
-    assert err.http_response.status_code == status
-    assert str(err) == msg
+    err_value = err.value
+    assert isinstance(err_value, exceptions.FirebaseError)
+    assert err_value.code == code
+    assert err_value.http_response is not None
+    assert err_value.http_response.status_code == status
+    assert str(err_value) == msg
 
 
-def instrument_mlkit_service(app=None, status=200, payload=None):
+def instrument_mlkit_service(app=None, status=200, uri=None, payload=None):
     if not app:
         app = firebase_admin.get_app()
     mlkit_service = mlkit._get_mlkit_service(app)
     recorder = []
-    mlkit_service._client.session.mount(
-        'https://mlkit.googleapis.com',
-        testutils.MockAdapter(payload, status, recorder)
-    )
+    if uri is None or uri is 'projects':
```

Comment:
May be change `uri` to a boolean `operations=False` parameter.

Or add an else clause below and raise an exception.

Reply:
done